book/moving_around.md

```diff
@@ -1,41 +1,104 @@
 
 ## Glob patterns (wildcards)
 
-The asterisk (\*) in the above optional argument "\*.md" is sometimes called a wildcard or a glob. It lets us match anything. You could read the glob "\*.md" as "match any filename, so long as it ends with '.md' "
+The asterisk (`*`) in the above optional argument `*.md` is sometimes called a wildcard or a glob. It lets us match anything. You can read this glob `*.md` as _"match any filename, so long as it ends with '.md'."_
 
 The most general glob is `*`, which will match all paths. More often, you'll see this pattern used as part of another pattern, for example `*.bak` and `temp*`.
 
-In Nushell, we also support double `*` to talk about traversing deeper paths that are nested inside of other directories. For example, `ls **/*` will list all the non-hidden paths nested under the current directory.
+Nushell also supports a double `*` which will traverse paths that are nested inside of other directories. For example, `ls **/*` will list all the non-hidden paths nested under the current directory.
+
+```nu
+ ls **/*.md
+╭───┬───────────────────────────────┬──────┬──────────┬──────────────╮
+│ # │             name              │ type │   size   │   modified   │
+├───┼───────────────────────────────┼──────┼──────────┼──────────────┤
+│ 0 │ CODE_OF_CONDUCT.md            │ file │  3.4 KiB │ 5 months ago │
+│ 1 │ CONTRIBUTING.md               │ file │ 11.0 KiB │ a month ago  │
+│ 2 │ README.md                     │ file │ 12.0 KiB │ a month ago  │
+│ 3 │ SECURITY.md                   │ file │  2.6 KiB │ 5 hours ago  │
+│ 4 │ benches/README.md             │ file │    249 B │ 2 months ago │
+│ 5 │ crates/README.md              │ file │    795 B │ 5 months ago │
+│ 6 │ crates/nu-cli/README.md       │ file │    388 B │ 5 hours ago  │
+│ 7 │ crates/nu-cmd-base/README.md  │ file │    262 B │ 5 hours ago  │
+│ 8 │ crates/nu-cmd-extra/README.md │ file │    669 B │ 2 months ago │
+│ 9 │ crates/nu-cmd-lang/README.md  │ file │  1.5 KiB │ a month ago  │
+╰───┴───────────────────────────────┴──────┴──────────┴──────────────╯
+```
+
+Here, we're looking for any file that ends with ".md". The double-asterisks further specify _"in any directory starting from here."_
+
+Nushell's globbing syntax not only supports `*`, but also matching [single characters with `?` and character groups with `[...]`](https://docs.rs/nu-glob/latest/nu_glob/struct.Pattern.html).
+
+Escaping the `*`, `?`, and `[]` patterns works by enclosing them in a single-quoted, double-quoted, or
+[raw string](working_with_strings.md#raw-strings). For example, to show the contents of a directory named
+`[slug]`, use `ls "[slug]"` or `ls '[slug]'`.
+
+However, _backtick_ quoted strings do not escape globs. For example:
+
+```nu
+# Removes all files in the current directory that contain
+# "myfile" as part of the filename
+> rm *myfile*
+
+# Quoted asterisk - Removes only a single file with the
+# name *myfile* (including the asterisks)
+> rm "*"
```

Comment:
I thought double quotes meant this is a literal and don't expand it. At least that's what I thought it was supposed to be. 

Example:
```nushell
❯ touch "$"
❯ ls
╭─#─┬─name─┬─type─┬─size─┬─modified─╮
│ 0 │ $    │ file │  0 B │ now      │
╰───┴──────┴──────┴──────┴──────────╯
❯ rm $
Error: nu::parser::variable_not_found

  × Variable not found.
   ╭─[entry #17:1:4]
 1 │ rm $
   ·    ┬
   ·    ╰── variable not found. Did you mean '$r'?
   ╰────

❯ rm "$"
```

Reply:
Thanks for catching that!

It looks like I was mid-edit on changing the `"*"` example to `"*myfile"`* and forgot to finish it. :-/

I've fixed the issue, but I've also expanded the text a bit.  Hope that looks better.